Mage/src/main/java/mage/constants/Layer.java

```diff
@@ -5,6 +5,7 @@
  * @author North
  */
 public enum Layer {
+    TransformEffects_0,
```

Comment:
This does not conform to 613.1 (interaction of continuous effects / layer system). If the workaround is necessary, needs clear explanation of the shortcomings of current mage engine implementation. Maybe related to 701.28 or 712.8e? Is the problem that defining TDFC characteristics shouldn't be a layer 1 effect in the first place?